Gruntfile.coffee

```diff
@@ -35,53 +36,55 @@ module.exports = ->
         files:
           './browser/noflo-html.min.js': ['./browser/noflo-html.js']
 
-    # Automated recompilation and testing when developing
-    watch:
-      files: ['spec/*.coffee', 'components/*.coffee']
-      tasks: ['test']
-
     # BDD tests on Node.js
-    cafemocha:
+    mochaTest:
       nodejs:
-        src: ['spec/*.coffee']
+        src: ['spec/*.coffee', 'test/*.coffee']
```

Comment:
It would probably be better to just move the mocha-ified tests to the `spec/` folder